storage/aptosdb/src/state_kv_db.rs

```diff
@@ -56,19 +57,13 @@ impl StateKvDb {
             });
         }
 
-        Self::open(
-            db_paths,
-            rocksdb_configs.state_kv_db_config,
-            readonly,
-            sharding,
-        )
+        Self::open(db_paths, rocksdb_configs.state_kv_db_config, readonly)
```

Comment:
let's rename to `Self::open_sharded()` or something?

Reply:
Done